### PR TITLE
How to Install Nagios 4.4.3 on CentOS 7

### PR DESCRIPTION
**Pull request by 1817666**
I have read and understood the Contributor's Certificate of Origin available at the end of
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: MohammadHossein Fakhraei eng.fakhraei@gmail.com

### CODE REVIEW

File: tutorials/how-to-install-nagios-on-centos-7/01.en.md [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+Then add the user to the group with these commands:
+
+```
+usermod -G nagcmd nagios
+usermod -G nagcmd apache
+```
+
+## Step 3 - Download Nagios Core and Nagios Plugin
+
+Download latest Nagios Core 4.3.4 and Nagios plugins 2.2.1 packages. Go to the [Nagios downloads page](http://www.nagios.org/download/core-stay-informed), and click the Skip to download link below the form. Copy the link address for the latest stable release so you can download it to your Nagios server.
```
Comment (n-se) [inline tutorials/how-to-install-nagios-on-centos-7/01.en.md]:
```suggestion
Download latest Nagios Core 4.3.4 and Nagios plugins 2.2.1 packages. Go to the [Nagios downloads page](https://www.nagios.org/download/core-stay-informed), and click the Skip to download link below the form. Copy the link address for the latest stable release so you can download it to your Nagios server.
```

Comment (n-se) [reply tutorials/how-to-install-nagios-on-centos-7/01.en.md]: Better use https by default. Also I don't get this form you are talking about with this link. Do you really get it?

File: tutorials/how-to-install-nagios-on-centos-7/01.en.md [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+# Just one contact defined by default - the Nagios admin (that's you)
+# This contact definition inherits a lot of default values from the
+# 'generic-contact' template which is defined elsewhere.
+
+define contact {
+
+    contact_name            nagiosadmin             ; Short name of user
+    use                     generic-contact         ; Inherit default values from generic-contact template (defined above)
+    alias                   Nagios Admin            ; Full name of user
+    email                   eng.fakhraei@gmail.com  ; <<***** CHANGE THIS TO YOUR EMAIL ADDRESS ******
```
Comment (n-se) [inline tutorials/how-to-install-nagios-on-centos-7/01.en.md]:
```suggestion
    email                   admin@example.com  ; <<***** CHANGE THIS TO YOUR EMAIL ADDRESS ******
```

Comment (1817666) [reply tutorials/how-to-install-nagios-on-centos-7/01.en.md]: Edited.

File: tutorials/how-to-install-nagios-on-centos-7/01.en.md (hunk 1 above)
Comment (1817666) [reply tutorials/how-to-install-nagios-on-centos-7/01.en.md]: Edited.

**Comment by 1817666** [comment]
Added `bash` behind all three `.

**Pull request merged**